### PR TITLE
Ds 106 modal responsive design

### LINKED ISSUE #106: style modal for mobile responsive
**Is your feature request related to a problem? Please describe.**
make pokeapp modal viewable on responsive screen sizes

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
_List Changes Introduced by this PR_
1. make mobile responsive design for modal in tablet
2. responsive modal for smartphone sizes

Closes #106

### CODE REVIEW

File: src/styles/components/_modal.scss [hunk 1]
```diff
@@ -245,3 +245,184 @@ $modal-padding: 25px;
 .aboutSpecsLabel {
   margin-right: 15px;
 }
+
+
+// ***** RESPONSIVE STYLING *****
+
+// ipad pro landscape
```
Comment (BlackCubes) [inline src/styles/components/_modal.scss:252]: I like how much more detailed you're getting with landscape vs. portrait :O

File: src/styles/components/_modal.scss [hunk 2]
```diff
@@ -245,3 +245,184 @@ $modal-padding: 25px;
...
+  width: 100%;
+  margin: 0 auto;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+
+}
+
+// iphone screen width
```
Comment (BlackCubes) [inline src/styles/components/_modal.scss:408]: Nice on making it work for very small screens 🙌

**Review by BlackCubes — APPROVED** [review]
Everything looks beautiful!

**Review by connielion — APPROVED** [review]
I see the comments and was wondering if these media queries takes care of Android devices too? Looks good so far otherwise!

**Pull request merged**